esy/Package.ml

```diff
@@ -53,6 +53,34 @@ module CommandList = struct
 
 end
 
+(**
+ * Scripts with keys as specified in "scripts".
+ *)
+module Scripts = struct
+
+  type script = {
+    key : string;
```

Comment:
Let's call it `name` instead? "Script name" sounds more right than "script key".

Reply:
Disregard this if we are going to make `t` be a map instead of a list.